src/sched_rebuild.c

```diff
@@ -59,6 +99,8 @@ void rebuild_sched_state(bool mod)
 		if (rq != task_rq(p))
 			continue;
 
+		switch_sched_class(p, mod);
```

Comment:
Seems you should switch back when clear_sched_state

Reply:
During clearing stage, it should use the callbacks of the current active scheduler.  So, there is no need to handle clear_sched_state.